openapi/components/schemas/RichBillingPortal.yaml

```
@@ -45,7 +46,7 @@ properties:
           type: string
           description: |-
             Status of the plan in the billing portal.
-            If status the is `archived`, customers can continue using the billing portal, 
+            If status the is `archived`, customers can continue using the billing portal,
```

nitpick: (not sure if this was a typo from one of my suggestions 😅)
```suggestion 
            If the status is `archived`, customers can continue using the billing portal,
```